### ISSUE TITLE
crontab can not run

### ISSUE THREAD

**Issue by bubifengyun** [yii2tech/crontab#7]
I find that the command will not execute .

I search and find that if I add this two lines in the head.

``` bash
SHELL=SHELL
PATH=/usr/bin:<and others>
yii crontab/index
```

it will run this command

``` bash
yii crontab/index
```

or, it will not run. others command will execute, such as `echo $PATH > /tmp/test`

so could you open the `applyLines()` as `public` for whom cannot execute the command?

thank you.

**Comment by bubifengyun** [comment]
sorry for my bad English.

I do as follows, run as root.

``` bash
crontab -u daemon -e
```

add this two lines copying from `/etc/crontab`.
save and exit, it will work well.

**Comment by klimov-paul** [comment]
Indeed crontab allows extra BASH settings at the top of the file.
Adding extra property to `CronTab` for such lines can be considered.

**Comment by bubifengyun** [comment]
thank you. I change protected to public, and add those two lines in the head. work well.

**Comment by klimov-paul** [comment]
Resolved by commit 7bad63ff6d06d10c3b880b18786b10b6eb368802
